dawn/main/ansible/Ansible.js

```diff
@@ -16,12 +19,31 @@ const StudentCodeStatus = DawnData.StudentCodeStatus;
 
 const runtimeBuilder = ProtoBuf.loadProtoFile(`${protoFolder}/runtime.proto`);
 const RuntimeData = runtimeBuilder.build('RuntimeData');
+const TCPData = runtimeBuilder.build('TCPData');
+
+let test = 0;
```

Comment:
My understanding of the doc you linked is that `let` in global scope does not bind the variable to the *global object*. Note that all properties of the global object are also global variables, but not all global variables are necessarily properties of the global object.

This distinction is minor, the only issue is if you try to do `global.test` (or `this.test` when this refers to `global`), but nobody ever does this.